python/cuml/test/test_array.py

```diff
@@ -97,9 +98,7 @@ def test_array_init(input_type, dtype, shape, order):
 
     assert ary.dtype == np.dtype(dtype)
 
-    if input_type == 'numpy':
-        assert isinstance(ary._owner, DeviceBuffer)
-    elif input_type in ['cupy', 'numba', 'series']:
+    if input_type in ['cupy', 'numba', 'series']:
```

Comment:
Do we need to add tests to explicitly check input from views? I don't see that explicit anywhere here but maybe it's implicit

Reply:
For views (and overall non contiguous inputs) this actually gets tested in test_input_utils. But also I'll be improving some things so I will make adding inputs from views tests more extensive as part of #2412 early next week